src/convertToObject.js

```diff
@@ -1,12 +1,26 @@
-'use strict';
+"use strict";
 
 /**
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const STYLES_ARRAY = sourceString.split(";");
+  const STYLES_OBJECT = {};
```

Comment:
these are NOT actually constant values, they depend on sourceString, and `{}` will be modified, please change those to `camelCase`